packages/bootstrap/scss/toolbar/_layout.scss

```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/toolbar/_layout.scss";
+
+
+.k-toolbar {
```

Comment:
Do we need this code? I mean should there be a border in the Bootstrap theme?
I couldn't find a design for it.

If there should be a border, then the code should be wrapped in a `@include exports("toolbar/layout/bootstrap") { }` block.

Reply:
I followed the design for the Editor toolbar